src/main.cc

```diff
@@ -134,7 +135,7 @@ static std::pair<struct tm, int> parse_time(const std::string &time_string) {
 int main(int argc, char **argv) {
   // TODO: use a real logging library
   CLI::App app{"Tracer"};
-
+  LoadMeshesFromOBJ("cube.obj");
```

Comment:
Stray debugging/testing code? Please drop.

Reply:
We should probably do one of the following in order to expose this.

1. Drop `--gltf_path` and replace it with a more generic `--scene_path` that calls `LoadMeshesFromGLTF()` if the file ends with `.gltf` or `.glb` and `LoadMeshesFromOBJ()` if it ends with `.obj`
2. Add a new flag called `--obj_path` and place it in a group with `--gltf_path` and require that only one of them be set.

I think I prefer (1). What do you think?